scripts/assets/file_content_validation_checks.textproto

```diff
@@ -465,3 +465,13 @@ file_content_checks {
   exempted_file_name: "scripts/src/javatests/org/oppia/android/scripts/regex/RegexPatternValidationCheckTest.kt"
   exempted_file_name: "utility/src/main/java/org/oppia/android/util/parser/image/UrlImageParser.kt"
 }
+file_content_checks {
+  filename_regex: ".+.xml"
+  prohibited_content_regex: "^\\s*<meta-data\\s+android:name\\s*=\\s*"firebase_analytics_collection_deactivated"\\s*android:value\\s*=\\s*"false".+$"
+  failure_message: "Firebase analytics collection should always be deactivated in develop"
```

Comment:
Nit: please end with punctuation (ditto for the error below) for consistency with other errors.

Reply:
Done